src/components/LayerCollection/LayerPOLine.js

```diff
@@ -184,13 +190,61 @@ function LayerPOLine({
     }
   };
 
+  const validateDuplicateLines = useCallback(
```

Comment:
can it be moved from component?

Reply:
it should + we have a ticket to use this one in open order functionality. Thanx